README.md

```diff
@@ -4,103 +4,99 @@ This package provides a set of language helpers, utility functions, and classes
 
-## Features
+## Installation
 
-### Feature Detection
+This package is currently in Alpha with a initial stable release scheduled for later this year. You can install the Alpha
+through npm:
 
-Using the latest Web technologies isn't always as straightforward as developers would like due to differing support
-across platforms. `dojo-core/has` provides a simple feature detection API that makes it easy to detect which platforms
-support which features.
+```
+npm install dojo@2.0.0-alpha4
+```
+
+You can also clone or download this repository.
 
-#### Detecting Features
+## Dojo Usage
 
-The default export of `dojo-core/has` is a function which accepts a single parameter: the name of the feature to test for.
-If the feature is available, a truthy value is returned, otherwise a falsy value is returned:
+### npm
+
+If using npm, you can access modules by using ```require``` like so:
 
 ```ts
-if (has('dom-addeventlistener')) {
-    element.addEventListener('click', function () { /* ... */ });
-}
+import lang = require('dojo/lang');
 ```
 
-#### Adding Feature Detections
-
-It's important to be able to add new feature tests that aren't provided out-of-the-box by `dojo-core/has`.
-This can be done easily by using the `add` function exported by the `has` module. It accepts two parameters:
-the name of the feature, and either an immediate value indicating its availability or a function that resolves to a
-value.
+### Downloading
 
-When a function is passed, the feature will be lazily evaluated - i.e. the function is not executed until the feature is
-actually requested. The return value is then cached for future calls for the same feature.
+To access modules use after cloning or downloading the repository, you can reference it by:
 
 ```ts
-import { add as hasAdd } from 'dojo-core/has';
-hasAdd('dom-queryselector', 'querySelector' in document && 'querySelectorAll' in document);
+import * as lang from 'src/lang'; // this imports all exports of the module as the object lang
 
-// Lazily executed; useful if a polyfill is loaded after page load
-hasAdd('typedarray', function () {
-    return 'ArrayBuffer' in window;
-});
+import { lateBind, mixin } from 'src/lang'; // this imports lateBind and mixin from the module
 ```
 
-#### Accessing the Feature Cache
+#### Compile To JavaScript
 
-`dojo-core/has` maintains object hashes containing keys that correspond to all features that have been both
-registered _and_ requested. The value associated with each feature name key corresponds to that feature's availability
-in the current environment. The object hash containing evaluated features is accessible via the `cache` export.
+Once downloaded, you can compile the TypesScript files by first installing the project dependencey with:
+
+```
+npm install
+```
+
+The by running this command:
+
+```
+
+node -e "require('grunt').tasks(['dev']);"
+
+```
+
+This will run the grunt 'dev' task.
+
+## Features
+
+### [Feature Detection](docs/has.md)
```

Comment:
Rather than linking the heading, mention the `has` module in the section and make that a link.